content/en/docs/concepts/policy/node-resource-managers.md

```diff
@@ -200,8 +200,8 @@ listed in alphabetical order:
 : Spread CPUs across different NUMA domains, aiming for an even balance between the selected domains
   (available since Kubernetes v1.23)
 
-`full-pcpus-only` (beta, visible by default)
-: Always allocate full physical cores (available since Kubernetes v1.22)
+`full-pcpus-only` (GA, visible by default)
+: Always allocate full physical cores (available since Kubernetes v1.22, GA since Kubernetes v1.32)
```

Comment:
Why v1.32? This should be v1.33...

Reply:
copy/paste mistake. You are obviously correct, fixed. Thanks for pointing it out.